packages/dictionary/data/chords.json

```diff
@@ -1,21 +1,22 @@
 {
   "4": ["1P 4P 7m 10m", ["quartal"]],
-  "64": ["5P 8P 10M"],
   "5": ["1P 5P"],
-  "M": ["1P 3M 5P", ["Major", ""]],
-  "M#5": ["1P 3M 5A", ["augmented", "maj#5", "Maj#5", "+", "aug"]],
+  "7": ["1P 3M 5P 7m", ["Dominant", "Dom"]],
+  "9": ["1P 3M 5P 7m 9M", ["79"]],
+  "11": ["1P 5P 7m 9M 11P"],
+  "13": ["1P 3M 5P 7m 9M 13M", ["13_"]],
+  "64": ["5P 8P 10M"],
+  "M": ["1P 3M 5P", ["Major", "maj", "", "Maj"]],
```

Comment:
I know "Major" is in the original json, but now I'm reviewing the changes, I think the same criteria as "minor" applies: normally is the name of a scale. I would remove them. What do you think?

Reply:
Yeah I was just being safe by not removing existing values, but I doubt anyone is depending on that nomenclature just for those two chords. Probably fine to remove them and "minor7"

What do you think about the change I mention above regarding changing the key "Maj7" to "M7". As with "major/minor", I doubt anyone is depending on that key just for M7 chords.

My implementation generates chords based off of the JSON keys, and that one does not follow the pattern of the rest.

Reply:
Hi @JakeIwen,

I don't understand what you mean by changing the key from "Maj7" to "M7" (to follow the object-key pattern?). Can you explain a little?

Anyway, I would prefer to use "Maj7" as default instead of "M7" because I think is clear.

Reply:
Sounds good about not adding not adding "minor" and keeping "Major" (and "minor7") until 2.0.
The following are the objcet-keys for "M" chords - effectively the default name for the chord.
The pattern i speak of is to begin them with a single character "M". 
One of these does not look like the rest. 
Maybe that could be modified in 2.0 as well?
Your call. 
```
"M":
"M#5":
"M#5add9":
"M13":
"M13#11":
"M6":
"M6#11":
"M69":
"M69#11":
"M7#11":
"M7#5": 
"M7#5sus4":
"M7#9#11":
"M7add13":
"M7b5":
"M7b6":
"M7b9":
"M7sus4":
"M9":
"M9#11":
"M9#5":
"M9#5sus4":
"M9b5":
"M9sus4":
"Madd9":
"Maj7":
"Mb5":
"Mb6":
"Msus2":
"Msus4":
"Maddb9":
```